doc/ha/overview.md

```diff
@@ -1,42 +1,229 @@
-# HA Setup for Development
+# OpenZiti HA
 
-**NOTE: HA is a work in progress and not yet usable for anything other than developing HA**
+This document gives a brief overview of how OpenZiti HA works and how it differs from running
+OpenZiti in non-HA mode.
 
```

Comment:
Maybe add a note about certs, why to use different intermediates, spiffe ids, what's expected in the cert chain, etc

Reply:
done